htdocs/societe/paymentmodes.php

```diff
@@ -1762,6 +1762,10 @@
 		dol_print_error($db);
 	}
 
+	//Hook to display your print listing (list of CB card from Stancer Plugin for example)
+	$parameters = array('arrayfields'=>array(), 'param'=>'', 'sortfield'=>'', 'sortorder'=>'', 'linetype'=>'');
```

Comment:
Hmmm,
there are lot of consequences and i can't store stancer stuff into stripe fields (just a question about "what if" i have stancer and stripe together).

then, i've just added two fields into that table to store stancer stuff and now i really need a hook to display data into that tab

but it could be the same problem for all others payment provider (sumup ...).